addons/mail/static/src/core/common/message_service.js

```diff
@@ -30,6 +30,9 @@ export class MessageService {
         if (convertBrToLineBreak(message.body) === body && attachments.length === 0) {
             return;
         }
+        await Promise.all(message.mentionedChannelPromises).then((fetchedChannels) => {
```

Comment:
`await ... then` should be replaced by `const ... = await ...; code on next line`.